src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ScannableItemTest.java

```diff
@@ -60,4 +70,10 @@ public void should_update_document_url_of_scannable_item() throws IOException {
             assertThat(item.getDocumentUrl()).isEqualTo("localhost/document/" + item.getId())
         );
     }
+
+    private Envelope getEnvelope() throws IOException {
```

Comment:
Have you considered this moving to a test helper or something instead of repeating this code ?

Reply:
I did. Perhaps this PR is the best place for it